Add Identity and Access Management into HCC settings procedure. (#642)
redhat-developer/app-services-guides: docs/registry/access-mgmt-registry/README.adoc

.Procedure

> . At the top of the {service-url-registry}[{registry} web console], select the gear icon. 

Reviewer: ```suggestion
. In the toolbar of the {service-url-registry}[{registry} web console], select the gear icon. 
```

> . Click *Settings*, then *Identity & Access Management*, then *User Access*, and then *Groups*.

Reviewer: ```suggestion
. Click *Settings* > *Identity & Access Management* > *User Access* > *Groups*.
```
